### PR TITLE
Pluralism

### PR DESCRIPTION
**Pull request by pesarkhobeee**
I added more channel to the repository

### CODE REVIEW

File: channels.json5 [hunk 1]
```diff
@@ -514,4 +514,39 @@ Add another row at the end of the list, similar to the following object
...
+    "name": "Dev Quest",
+    "category": "computer",
+  },
+  {
+    "id": "UCvRY5AQuQy91hUhpRbJayvg",
+    "name": "Bazikhar Videocast",
+    "category": "misc",
+  },
+  {
+    "id": "UCdtGLDEOsnWlW1IsQFbCAsA",
```
Comment (omid) [inline channels.json5:533]: It's duplicated.
You can replace it with my channel :D

Comment (pesarkhobeee) [reply channels.json5:533]: OK :]

File: channels.json5 [hunk 2]
```diff
@@ -514,4 +514,39 @@ Add another row at the end of the list, similar to the following object
...
+  },
+  {
+    "id": "UC6qiCkKqaipJmS73UK1xHYw",
+    "name": "Saeed Shahrivari",
+    "category": "computer",
+  },
+  {
+    "id": "UC0IBNJgLL5zUHXUxazRSMZA",
+    "name": "Ashpazi Ba Sana",
+    "category": "misc",
```
Comment (omid) [inline channels.json5:545]: We have a "cooking" category, is it related to cooking? 🤔

Comment (pesarkhobeee) [reply channels.json5:545]: Well, it seems she is doing smocking stuff too :))
https://www.youtube.com/playlist?list=UUMO0IBNJgLL5zUHXUxazRSMZA

Comment (omid) [reply channels.json5:545]: :)))

**Pull request merged**